tests/platform_tests/daemon/test_chassisd.py

```diff
@@ -0,0 +1,211 @@
+"""
+Check daemon status inside PMON container. Each daemon status is checked under the conditions below in this script:
+* Daemon Running Status
+* Daemon Stop status
+* Daemon Restart status
+This script is to cover the test case in the SONiC platform daemon and service test plan:
+https://github.com/Azure/sonic-mgmt/blob/master/docs/testplan/PMON-Services-Daemons-test-plan.md
+"""
+import logging
+import re
+import time
+
+from datetime import datetime
+
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.daemon_utils import check_pmon_daemon_enable_status
+from tests.common.platform.processes_utils import wait_critical_processes, check_critical_processes
+from tests.common.utilities import compose_dict_from_cli, skip_release, wait_until
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2'),
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer
```

Comment:
This will disable sanity check and log analyzer permanently for this test. Can we use pytest options to disable/enable sanity check and log analyzer?

Reply:
Updated, please check